Makefile

```diff
@@ -8,10 +8,13 @@ test-coverage:
 	poetry run python -m coverage run -m unittest discover -s aptos_sdk/ -p '*.py' -t ..
 	poetry run python -m coverage report
 
+test-spec:
+	poetry run behave
+
 fmt:
-	find ./examples ./aptos_sdk . -type f -name "*.py" | xargs poetry run autoflake -i -r --remove-all-unused-imports --remove-unused-variables --ignore-init-module-imports
-	poetry run isort aptos_sdk examples
-	poetry run black aptos_sdk examples
+	find ./examples ./aptos_sdk ./features . -type f -name "*.py" | xargs poetry run autoflake -i -r --remove-all-unused-imports --remove-unused-variables --ignore-init-module-imports
+	poetry run isort aptos_sdk examples features
+	poetry run black aptos_sdk examples features
 
```

Comment:
add to lint below, please!

Reply:
I think I actually removed it from the lint, because something did not play well.  Let me try again.